### PR TITLE
Configure roctx for AMD kernel naming

### PR DESCRIPTION
**Pull request by artv3**
Add support for ROCTX to enable kernel naming with RAJA Teams.

### CODE REVIEW

**Comment by artv3** [comment]
Resolves https://github.com/LLNL/RAJA/issues/1118

File: CMakeLists.txt [hunk 1]
```diff
@@ -236,6 +237,12 @@ if (ENABLE_HIP)
   set(raja_depends
     ${raja_depends}
     hip)
+  if(ENABLE_ROCTX)
+    include(FindRoctracer)
```
Comment (davidbeckingsale) [inline CMakeLists.txt:241]: I think you should move the FindRoctracer call into `cmake/SetupPackages.cmake`

File: cmake/SetupPackages.cmake [hunk 2]
```diff
@@ -61,6 +61,9 @@ endif ()
 
 if (ENABLE_HIP OR ENABLE_EXTERNAL_ROCPRIM)
   find_package(RocPRIM)
+  blt_import_library(NAME roctx
```
Comment (davidbeckingsale) [inline cmake/SetupPackages.cmake:64]: I think this should be in a separate conditional block (not inside the "OR ENABLE_EXTERNAL_ROCPRIM") - it should be guarded with ENABLE_ROCTX.

**Review by davidbeckingsale — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (artv3) [reply CMakeLists.txt]: Done!

**Review by davidbeckingsale — APPROVED** [review]

**Review by tomstitt — APPROVED** [review]
Thank you @artv3 !

**Pull request merged**